### PR TITLE
groups: new mobile member list

### PR DESCRIPTION
**Pull request by patosullivan**
Fixes LAND-792

Looks like this:

https://github.com/tloncorp/landscape-apps/assets/1221094/2b47a184-5119-4eb3-bc56-2f30b67f0dea

### CODE REVIEW

File: ui/src/groups/Members.tsx [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+              {toggleSearch ? (
+                <XIcon className="h-6 w-6 text-gray-800" />
+              ) : (
+                <MagnifyingGlassMobileNavIcon className="h-6 w-6 text-gray-800" />
+              )}
+            </button>
+          </div>
+        }
+      />
+      <div className="h-full overflow-auto px-4">
```
Comment (jamesacklin) [inline ui/src/groups/Members.tsx:362]: Nice, looks good! We will need to virtualize this list, though - it takes ~40s to render all members of Tlon Local.

**Review by jamesacklin — APPROVED** [review]

**Review by jamesacklin — CHANGES_REQUESTED** [review]
Virtualize please

File: ui/src/groups/Members.tsx (hunk 1 above)
Comment (patosullivan) [reply ui/src/groups/Members.tsx:362]: will do, chef

**Comment by patosullivan** [comment]
@jamesacklin we good on this one?

**Pull request merged**